hypervisor/src/mshv/mod.rs

```diff
@@ -279,6 +279,13 @@ impl hypervisor::Hypervisor for MshvHypervisor {
     fn get_supported_cpuid(&self) -> hypervisor::Result<Vec<CpuIdEntry>> {
         Ok(Vec::new())
     }
+
+    /// Get maximum number of vCPUs
+    fn get_max_vcpus(&self) -> u32 {
+        // TODO: Using HV_MAXIMUM_PROCESSORS would be better
```

Comment:
@russell-islam @NunoDasNeves @jinankjain can you give an ack or nack on this hunk?

Should we fix the IOCTL API? IIRC there is a VM size that has more than 256 CPUs, which means the hypervisor definitely supports >256.

For the core scheduler, we want to query the hypervisor.

For the root scheduler, we _should_ be able to bump the limit all the way to what KVM can offer.

Reply:
I agree we should fix the ioctl API instead of hardcoding it to 256.